nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -693,48 +671,27 @@ define([
          * This doesn't change any state, just runs what it's told to,
          * and returns the widget's start() Promise.
          * @param {string} tab
-         * @param {string} fileType
          */
-        function runTab(tab, fileType) {
+        function runTab(tab) {
             tabWidget = tabSet.tabs[tab].widget.make({
                 bus: controllerBus,
                 cell,
-                fileType,
                 jobId: undefined,
                 jobManager,
                 model,
-                spec: specs[typesToFiles[state.fileType.selected].appId],
+                specs,
```

Comment:
We don't track the current file type (and app) here anymore, so just send all the specs and the types.